### DISCUSSION TITLE
what custom smtp service do you recommend?

### DISCUSSION THREAD

**Discussion by devhandler** [supabase/supabase#16201] [Questions]
now supabase email limit is down to 4. what custom smtp do you think are good for testing and initial product launch?

**Comment by Collin0810** [comment]
Resend might work for you! https://resend.com/docs/send-with-supabase-smtp

**Reply by devhandler** [reply]
yeah, i saw that. curious of the pros and cons vs mailchimp, amazon ses, sendgrid, etc. would like to see people's different opinions.
